src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let wasteStyles = sourceString.split(';');
+  const obj = {};
+
+  wasteStyles = wasteStyles.map((line) => line.trim());
+
+  for (const styles of wasteStyles.filter(Boolean)) {
+    const style = styles.split(':');
+    const key = style[0].trimStart().trimEnd();
+    const value = style[1].trimStart().trimEnd();
```

Comment:
Consider using `trim()` instead of `trimStart().trimEnd()` for both `key` and `value`. The `trim()` method is more concise and achieves the same result by removing whitespace from both ends of a string.